client/pwa/src/db.ts

```diff
@@ -40,10 +41,33 @@ interface Whoami {
   is_subscriber: boolean;
 }
 
+export enum ContentStatusPhase {
+  initial = "initial",
+  idle = "idle",
+  download = "download",
+  unpack = "unpack",
+  clear = "clear",
+}
+
+export interface ContentVersion {
+  version: string;
+  date: string;
+}
+
+export interface ContentStatus {
+  id?: number;
+  phase: ContentStatusPhase;
+  local: ContentVersion | null;
```

Comment:
nit: perhaps a little more verbose naming local_version, remote_version .

Reply:
Before there were two properties `local_version` and `local_date`, so I merged them into one `local` object. I'm not totally against `local_version.version`, but I like the aesthetics of `local.version === remote.version`.

What do you think?

Reply:
I have now renamed `ContentVersion` to `LocalStatus` and reused `UpdateData` as `RemoteStatus`.

Guess this makes more sense now:

- `contentStatus` = ContentStatus
- `contentStatus.local` = Local[Content]Status
- `contentStatus.remote` = Remote[Content]Status

Let me know what you think.

Reply:
I think that looks fine.